src/AzSearch.tsx

```diff
@@ -79,6 +81,15 @@ class Automagic {
             document.getElementById(htmlId)
         );
     }
+
+    public addFilterBar(htmlId: string) {
```

Comment:
Love the feature! Do you think we can come up with a more descriptive/self-documenting name for the component and api? Some ideas:

- ResetFiltersButton/addResetFiltersButton
- FiltersClearAllButton/addFiltersClearAllButton

Filter bar seems a little too generic unless we have plans to add a bunch more widgets/features to it.